pyproject.toml

```diff
@@ -0,0 +1,8 @@
+[build-system]
+requires = [
+    "setuptools>=20.5.0",
+    "wheel",
+    "cmake>=3.12",
+    "pip>=6.0.0",
```

Comment:
cmake doesn't need it.

Reply:
We originally needed to check setup-tools and pip versions because on _very_ ancient distributions, neither would actually do version checking for itself (meaning pip/setup-tools wouldn't detect if the version of setup-tools needed to be updated).  At this point I think we can drop the local requirement since those are prerequisites of installing via pip/setup.py itself... unless we're using features present in those libraries after that point.  At least at one point we were using stuff from "more modern" setup-tools.